pkg/middleware/scope_test.go

```diff
@@ -48,6 +52,11 @@ var _ = Describe("Scope Suite", func() {
 				Expect(scope).ToNot(BeNil())
 				Expect(scope.ReverseProxy).To(BeFalse())
 			})
+
+			It("sets the RequestID using the request", func() {
```

Comment:
Do we have a test when the request ID isn't already set?

Reply:
Added tests for no + custom request ID headers